subworkflows/local/variant_calling/call_sv_MT.nf

```diff
@@ -11,19 +11,24 @@ workflow CALL_SV_MT {
         ch_fasta        // channel: [mandatory] [ val(meta), path(fasta) ]
 
     main:
-        ch_versions = Channel.empty()
+        ch_versions       = Channel.empty()
+        ch_eklipse_del    = Channel.empty()
+        ch_eklipse_genes  = Channel.empty()
+        ch_eklipse_circos = Channel.empty()
 
-        EKLIPSE(ch_bam_bai,[])
+        if (!params.skip_eklipse){
+            EKLIPSE(ch_bam_bai,[])
+            ch_versions = ch_versions.mix(EKLIPSE.out.versions.first())
+       }
 
         MT_DELETION(ch_bam_bai, ch_fasta)
 
-        ch_versions = ch_versions.mix(EKLIPSE.out.versions.first())
         ch_versions = ch_versions.mix(MT_DELETION.out.versions.first())
 
     emit:
-        eklipse_del    = EKLIPSE.out.deletions         // channel: [ val(meta), path(csv) ]
-        eklipse_genes  = EKLIPSE.out.genes             // channel: [ val(meta), path(csv) ]
-        eklipse_circos = EKLIPSE.out.circos            // channel: [ val(meta), path(png) ]
+        eklipse_del    = ch_eklipse_del                // channel: [ val(meta), path(csv) ]
```

Comment:
will they always be empty or do I miss something here? Do you need to add the output from EKLIPSE in line 22 etc.?